### DISCUSSION TITLE
Trouble to connect Artisan with Berto D3

### DISCUSSION THREAD

**Discussion by darkslay3r** [artisan-roaster-scope/artisan#1708] [Q&A]
Hi, i am Indra and i am new here.

I have a problem with connecting Macbook Pro 2015 with latest Artisan 3.0.2 to Berto D3 today, previously i've been using it for 3 days just normal, and today i unable to connect it again.

My Berto D3 interface is FT232R USB UART.
I have installed all drivers from this link https://artisan-scope.org/modbus_serial/ and was detected by Artisan apps but unable to start with error message "Modbus Error : failed to connect".

Please help me :'(

**Comment by poundy** [comment]
I have no experience with Macs or with that roaster, but on PC there is always a chance the serial port your device presents to the computer changes designation. On a PC that's having Artisan confifgured to talk to COM4 but when you connect the cable to a different USB port it presents to the computer as COM9. I would be checking your ports to see what is showing; do an `ls /dev/tty*` and make sure you have that configured in Artisan

**Comment by MAKOMO** [comment]
Did the connection started to fail after upgrading Artisan to version 3.0.2 or did something else change after those 3 "normal" days?

**Reply by mxofelix** [reply]
i also have been facing the same problem, the MODBUS connection connect and disconnect occasionally after upgrading to 3.0.2, so i revert back to 3.0.0.
